CHANGELOG.md

```diff
@@ -75,6 +75,7 @@
 * [CHANGE] Deprecated `--rule-files` flag in favor of CLI arguments. #7756
 * [BUGFIX] Fix panic in `loadgen` subcommand. #7629
 * [BUGFIX] `mimirtool rules prepare`: do not add aggregation label to `on()` clause if already present in `group_left()` or `group_right()`. #7839
+* [ENHANCEMENT] Add flag `--output-dir` and `--save-file` to `mimirtool rules get` to allow persisting rule to a file for edit and re-upload. #7247
```

Comment:
I think it's correct now to remove reference to `--save-file`.

I also think it's more correct to call `--output-dir` an option rather than a flag.
Although flag is often used synonymously, I think it's original meaning was an option that toggled behavior, that is, that it took no arguments.

```suggestion
* [ENHANCEMENT] Add option `--output-dir` to `mimirtool rules get` to allow persisting rule to a file for edit and re-upload. #7247
```

Reply:
Ah.... great spot, you are absolutely correct that I forgot about this and only adjusted the doc.  I never knew there is a difference between option and flag so TIL.